src/components/SplitButton/SplitButton.module.css

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2023 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+.splitButton {
+  display: flex;
+}
+
+.mainActionButton {
+  border-top-right-radius: 0px;
+  border-bottom-right-radius: 0px;
+
+  &:hover {
+    z-index: 999;
```

Comment:
Can be centralized in zIndex in some way. I am scared to fill DS to micro values like 998, 997, etc... and lose control of them

Reply:
this only applies while actively hovering the item, which means it is already on top of anything else